superset-frontend/src/featureFlags.ts

```diff
@@ -38,6 +38,8 @@ export type FeatureFlagMap = {
 declare global {
   interface Window {
     featureFlags: FeatureFlagMap;
+    $: any;
+    jQuery: any;
```

Comment:
I'm _not_ worried about, this, and it's not worth spending any real time on, but curious if https://www.npmjs.com/package/@types/jquery might provide easy types for these